frontend/src/stores/dataTable.ts

```diff
@@ -0,0 +1,179 @@
+import { defineStore } from "pinia";
```

Comment:
This is unfortunate naming. xxxTable usually indicates a view component instead of a module component.